### ISSUE TITLE
Webpack 2: Installation of beta tag broken on NPM

### ISSUE THREAD

**Issue by ryanwilsonperkin** [webpack-contrib/extract-text-webpack-plugin#433]
Hello!

Just noticed that the recommended installation method for webpack (`npm install --save-dev extract-text-webpack-plugin@beta`) appears to be broken. Not super familiar with npm's version specifiers, but this could have to do with the v2.0.0 push that happened a few hours ago.

Perhaps the recommendation is just "update to 2.0.0 now"?
If so let me know and I'd be happy to submit a PR to update the docs.

OS: Mac OS X 10.11.6
Node: 4.7.3
NPM: 4.2.0

To reproduce:
```
$ npm install extract-text-webpack-plugin@beta
npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/ryan/.nvm/versions/node/v4.7.3/bin/node" "/Users/ryan/.nvm/versions/node/v4.7.3/bin/npm" "install" "extract-text-webpack-plugin@beta"
npm ERR! node v4.7.3
npm ERR! npm  v4.2.0
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: extract-text-webpack-plugin@beta
npm ERR! notarget Valid install targets:
npm ERR! notarget 0.1.0, 0.1.1, 0.1.2, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.5, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.3.6, 0.3.7, 0.3.8, 0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.7.1, 0.8.0, 0.8.1, 0.8.2, 0.9.0, 0.9.1, 1.0.0, 1.0.1, 2.0.0-beta.0, 2.0.0-beta.1, 2.0.0-beta.2, 2.0.0-beta.3, 2.0.0-beta.4, 2.0.0-beta.5, 2.0.0-rc.0, 2.0.0-rc.1, 2.0.0-rc.2, 2.0.0-rc.3, 2.0.0
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ryan/.npm/_logs/2017-02-24T20_16_50_334Z-debug.log
```

**Comment by d3viant0ne** [comment]
@ryanwilsonperkin - The as of this morning, the `beta` tag would have been a version behind `latest` which would have cause other issues.

The `beta` tag in npm no longer exists so yes, `npm i extract-text-webpack-plugin@latest --save` is the correct route.

**Comment by d3viant0ne** [comment]
In the next day or two, the `beta` tag will be used to published `v2.1.0-beta.0` which has a few pull request waiting to merge on the `2.0.0` release.

**Comment by ryanwilsonperkin** [comment]
Looks like @candu beat me to it in #432
Thanks! We'll update our `package.json`
